Cargo.toml

```
@@ -32,7 +32,7 @@ inventory = { version = "0.3.5", optional = true }
 
 # crate integrations that can be added using the eponymous features
 anyhow = { version = "1.0.1", optional = true }
-bigdecimal = { version = "0.4", optional = true }
+bigdecimal = { version = "0.4.3", optional = true }
```

```suggestion
bigdecimal = { version = "0.4.4", optional = true }
```

`0.4.4` reverted some formatting behavior that changed in `0.4.3`, and it's strongly preferred.

I don't know what the policy is on pinning dependencies, but any `>= 0.4.4, < 0.5` should work.

---

I picked `0.4.3` because it was the lowest version with `to_scientific_notation`. But now that I do not use that function, which version bound do you recommend?

---

Looks like `into_bigint_and_scale` is not available on 0.4.5. but `into_bigint_and_exponent` is. Are they the same in all versions?

---

I would just limit the version to where the function is available. Since which version is it guaranteed to exist?